host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake

```diff
@@ -0,0 +1,10 @@
+##############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+# project contributors. See the COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (MIT)
+##############################################################################
+set(COMMON_FLAGS "-gxx-name=/usr/tce/packages/gcc/gcc-7.1.0/bin/g++ -std=c++11")
```

Comment:
icpc_19_0_4_gcc8headers.cmake suggests that gcc 8 headers are used, which is the case in the toolchain host-config file. Overriding this here is not the intended way to do things. A new toolchain should be created:
icpc_19_0_4_gcc7headers.cmake.

Reply:
I was matching the naming of the configuration in radiuss-ci, but you are right - this is a stupid name.

In the radiuss-ci host config, common flags are defined, but not added anywhere: https://github.com/LLNL/radiuss-ci/blob/master/gitlab/conf/host-configs/toss_3_x86_64_ib/icpc_19_0_4_gcc8headers.cmake